README.md

```diff
@@ -55,6 +58,43 @@ Options:
 
 - `kubectl`: Required for interacting with Kubernetes clusters.
 
+### IRSA Configuration Check (`/checks/kube/aws-irsa.sh`)
```

Comment:
I'd add somewhere that a Helm chart release install is a requirement for this script.
E.g. `helm template` + `kustomize` or `ArgoCD` (uses helm template) would not result in Helm chart releases.